### PR TITLE
rename LICENCE to LICENSE

### PR DESCRIPTION
**Pull request by Kadantte**

### CODE REVIEW

**Comment by Kadantte** [comment]
Because if you use this link you will have Error 404...
```
https://github.com/fateloli/nhitomi/blob/master/LICENCE
```
So the best way is to change it to
```
https://github.com/fateloli/nhitomi/blob/master/LICENSE
```
image

File: README.md [hunk 1]
```diff
@@ -38,4 +38,4 @@ Then run the following commands:
 
 ## Licence
```
Comment (phosphene47) [inline README.md]: what about this line?

Comment (Kadantte) [reply README.md]: I think we are ready now

File: README.md [hunk 2]
```diff
@@ -36,6 +36,6 @@ Then run the following commands:
 2. `dotnet build` — builds the bot.
 3. `dotnet run` — runs the bot.
 
-## Licence
+## License
 
-This project is licensed under the [MIT licence](https://opensource.org/licenses/MIT). Please see the [licence](LICENCE) for more information.
```
Comment (phosphene47) [inline README.md]: also `MIT license`

**Pull request merged**